services/harmony/app/util/parameter-parsers.ts

```diff
@@ -95,10 +110,22 @@ export function handleScaleExtent(
  */
 export function handleScaleSize(
   operation: DataOperation,
-  query: Record<string, number[]>): void {
+  query: Record<string, number[]>): void;
+export function handleScaleSize(
+  operation: DataOperation,
+  query: Record<string, string>): void;
+export function handleScaleSize(
+  operation: DataOperation,
+  query: Record<string, number[] | string>): void {
   if (query.scalesize) {
-    const [x, y] = query.scalesize;
-    operation.scaleSize = { x, y };
+    if (typeof query.scalesize === 'string') {
+      const scaleSizeString: string = query.scalesize.replace('(', '').replace(')', '');
```

Comment:
Similar story to `scaleextent`; the string passed around for `scalesize` was something like `(123,456)`.